massgov/openmass
# Update new/activated user email message content

**Description:**
Update content of emails sent to users.


**Jira:**
https://jira.mass.gov/browse/DP-17990


**To Test:**
Test this on local so you have easy access to mailhog (just go to mailhog.mass.local in a different tab & voila, you'll see any mail sent by your local environment).
- [ ] Create a new user, making sure to check "notify user"
- [ ] Open the email in mailhog.mass.local and confirm all links work
- [ ] Activate a blocked user
- [ ] Open the email in mailhog.mass.local and confirm the content matches that of the new user (except the very first sentence, which is a tiny bit different) -- and that all the links work

**Screenshots/GIFs:**
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.

---

[Peer Review Checklist](https://github.com/massgov/opemass/blob/develop/docs/peer_review_checklist.md)
